common/xfer/controls.go

```diff
@@ -18,10 +18,15 @@ type Request struct {
 
 // Response is the Probe -> App -> UI message type for the control RPCs.
 type Response struct {
-	Value  interface{} `json:"value,omitempty"`
-	Error  string      `json:"error,omitempty"`
-	Pipe   string      `json:"pipe,omitempty"`
-	RawTTY bool        `json:"raw_tty,omitempty"`
+	Value interface{} `json:"value,omitempty"`
+	Error string      `json:"error,omitempty"`
+
+	// Pipe specific fields
+	Pipe   string `json:"pipe,omitempty"`
+	RawTTY bool   `json:"raw_tty,omitempty"`
+
+	// Remove specific fields
+	RemovedNode string `json:"removedNode,omitempty"` // Set if node was removed
```

Comment:
A node being in `remove` does not necessarily mean that it was *removed*. Correct me if I'm wrong, but it could have other reasons for short dis-/re-appearance.